daemon/api.go

```diff
@@ -281,6 +281,11 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 		includeLocal = true
 	}
 
+	var includeTypes []string
+	if len(query["types"]) > 0 {
```

Comment:
There are two ways of doing these: either we have a singular `type` option and accept multiple, or we have a plural `types` which can be comma-separated and only use the first one. We've done the latter for `sources`, but it'd be easy to change. I think the comma-separated one is slightly more dev-friendly, but the other is more correct.